pages/resultset.py

```diff
@@ -12,6 +12,10 @@
 
 class ResultsetPage(Page):
 
+    _page_title = '[0] mozilla-inbound'
+
+    _first_resultset_locator = (By.CSS_SELECTOR, '.th-view-content > ng-view:nth-child(1) > div:nth-child(2) > div:nth-child(1) > span:nth-child(1) > span:nth-child(1) > span:nth-child(2) > a:nth-child(1)')
+    _first_resultset_number_locator = (By.CSS_SELECTOR, '.th-view-content > ng-view:nth-child(1) > div:nth-child(2) > div:nth-child(3) > div:nth-child(2) > span:nth-child(1) > ul:nth-child(1) > li:nth-child(2) > span:nth-child(1) > span:nth-child(1) > a:nth-child(1)')
```

Comment:
This isn't really a number, but a revision. Also, this is a rather complex locator. Looking at the DOM, I think this would be suitable: `.result-set .revision-holder a`.

Also, this locator isn't specific to the first match, so we don't need to prefix it with `first_`, we just only use the first match in the test/page object.